README.md

```diff
@@ -159,8 +174,8 @@ The model artifact must have a **jar** extension.
 Inference models support unstructured mode, where input and output are not verified and can be almost anything.
 This is your responsibility to verify correctness.
 
-### Custom hooks for Python and R models
-Include any necessary hooks in a file called `custom.py` for Python models or `custom.R` for R models alongside your model artifacts in your model folder:
+### Custom hooks for Python, R models, and Julia models
```

Comment:
Probably there should be 3 `models` words or just 1.

Reply:
FIXED